### DISCUSSION TITLE
Can't install in windows10

### DISCUSSION THREAD

**Discussion by fangod123** [Difegue/LANraragi#600] [Q&A]
I download latest release lanraragi.msi and when I install for my PC for many times but this window show up always.
![123](https://user-images.githubusercontent.com/95561663/161679906-11ad9663-0e50-453d-93a3-0c8cd42f1698.PNG).

**Comment by fangod123** [comment]
windows version:
版本	Windows 10 企业版
版本号	20H2
安装日期	‎2022/‎2/‎10
操作系统内部版本	19042.1586
体验	Windows Feature Experience Pack 120.2212.4170.0

**Comment by Difegue** [comment]
Please check the troubleshooting instructions at https://sugoi.gitbook.io/lanraragi/v/dev/installing-lanraragi/windows#troubleshooting first; The msi log should be helpful.
